suites/squid/cephfs/tier-2_cephfs_test-volume-management_arch.yaml

```diff
@@ -358,3 +359,9 @@ tests:
       polarion-id: CEPH-83604184
       desc: cephfs subvolume idempoence earmark
       abort-on-fail: false
+  - test:
```

Comment:
The test needs to be in the suite : teir-2_file-dir-lay_vol-mgmt_nfs.yaml
Here, please note the tier-2_cephfs_test-volume-management_arch.yaml file is a archive file and is merged in to the suite file : teir-2_file-dir-lay_vol-mgmt_nfs.yaml
Please fix the "tier" in the suite file when merging the test.

Reply:
Fixed